File: mobile/src/main/javaBackup/com/planit/home/planit007/data/DbHelpers/GroupDbHelper.java
package planit007.planit.home.planit007.data.DbHelpers;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

import com.planit.mobile.data.Contracts.GroupContract;

/**
 * Created by Home on 2017-08-02.
 */

public class GroupDbHelper extends SQLiteOpenHelper{

    static final String DATABASE_NAME = "groups.db";

    static final int DATABASE_VERSION = 15;

    public GroupDbHelper (Context context) {

        super(context, DATABASE_NAME, null, DATABASE_VERSION);
    }

    @Override
    public void onCreate(SQLiteDatabase sqLiteDatabase) {

        final String CREATE_GROUP_TABLE = "CREATE TABLE " +
                GroupContract.GroupEntry.TABLE_NAME + " (" +
                GroupContract.GroupEntry._ID + " INTEGER PRIMARY KEY AUTOINCREMENT, " +
                GroupContract.GroupEntry.COLUMN_GROUP_NAME + " TEXT NOT NULL " +
                ");";

        sqLiteDatabase.execSQL(CREATE_GROUP_TABLE);
    }

    @Override
    public void onUpgrade(SQLiteDatabase sqLiteDatabase, int i, int i1) {

        sqLiteDatabase.execSQL("DROP TABLE IF EXISTS " + GroupContract.GroupEntry.TABLE_NAME);

        onCreate(sqLiteDatabase);
    }
}

File: mobile/src/main/kotlin/com/planit/mobile/data/DbHelpers/GroupDbHelper.kt
package com.planit.mobile.data.DbHelpers

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

import com.planit.mobile.data.Contracts.GroupContract

/**
 * Created by Home on 2017-08-02.
 */

class GroupDbHelper(context: Context) : SQLiteOpenHelper(context, DATABASE_NAME, null, DATABASE_VERSION) {

    override fun onCreate(sqLiteDatabase: SQLiteDatabase) {

        val CREATE_GROUP_TABLE = "CREATE TABLE " +
                GroupContract.GroupEntry.TABLE_NAME + " (" +
                GroupContract.GroupEntry._ID + " INTEGER PRIMARY KEY AUTOINCREMENT, " +
                GroupContract.GroupEntry.COLUMN_GROUP_NAME + " TEXT NOT NULL " +
                ");"

        sqLiteDatabase.execSQL(CREATE_GROUP_TABLE)
    }

    override fun onUpgrade(sqLiteDatabase: SQLiteDatabase, i: Int, i1: Int) {

        sqLiteDatabase.execSQL("DROP TABLE IF EXISTS " + GroupContract.GroupEntry.TABLE_NAME)

        onCreate(sqLiteDatabase)
    }

    companion object {

        internal val DATABASE_NAME = "groups.db"

        internal val DATABASE_VERSION = 16
    }
}